app/components/forms/wizard/basic-details-step.js

```diff
@@ -106,6 +106,12 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     return filter(this.get('data.event.tickets').toArray(), ticket => ticket.get('type') === 'paid').length > 0;
   }),
 
+  hasDonationTickets: computed('data.event.tickets.[]', {
+    get() {
```

Comment:
Why do you need a explicit getter if there is no setter ?

Reply:
@niranjan94, Traditional @computed seems no good here, So read about this in the docs.
This was the with the decorator, But seems not working, Any mistake?
```js
@computed('data.event.tickets.[]')
  get hasDonationTickets() {
    return filter(this.get('data.event.tickets').toArray(), ticket => ticket.get('type') === 'donation').length > 0;
  },
```

Reply:
@niranjan94, Traditional @computed seems no good here, So read about this in the docs.
This was the with the decorator, But seems not working, Any mistake?
```js
@computed('data.event.tickets.[]')
  get hasDonationTickets() {
    return filter(this.get('data.event.tickets').toArray(), ticket => ticket.get('type') === 'donation').length > 0;
  },
```

Reply:
You cannot use a decorator on a non-class component. 

For this you should do it the old way ...

```js
hasDonationTickets: computed('data.event.tickets.[]', function() {
	return condition
})
```